sources/packages/backend/apps/api/src/route-controllers/audit/_tests_/audit.controller.e2e-spec.ts

```diff
@@ -0,0 +1,117 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getAESTToken,
+  getInstitutionToken,
+  getStudentToken,
+  InstitutionTokenTypes,
+} from "../../../testHelpers";
+import { getProviderInstanceForModule } from "@sims/test-utils";
+import { AppModule } from "../../../app.module";
+import { AuditEvent, AuditService } from "../../../services";
+
+describe("AuditController(e2e)-audit", () => {
+  let app: INestApplication;
+  let auditService: AuditService;
+
+  beforeAll(async () => {
+    const { nestApplication, module } = await createTestingAppModule();
+    app = nestApplication;
+    auditService = await getProviderInstanceForModule(
+      module,
+      AppModule,
+      AuditService,
+    );
+    auditService.logger.log = jest.fn();
+  });
+
+  it(`Should log 'Logged In' message when ${AuditEvent.LoggedIn} is requested.`, async () => {
+    // Arrange
+    const endpoint = `/audit/${AuditEvent.LoggedIn}`;
+    const token = await getAESTToken(AESTGroups.Operations);
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.CREATED);
+
+    expect(auditService.logger.log).toHaveBeenCalledWith(
```

Comment:
The mock should be reset in the `beforeEach` otherwise the test result will be misleading.